cudaproxy/src/cudaproxy.rs

```diff
@@ -173,4 +389,126 @@ pub extern "C" fn cudaMemcpy(
     return unsafe {
         syscall5(SYS_PROXY, ProxyCommand::CudaMemcpy as usize, dst as * const _ as usize, src as usize, count as usize, kind as usize) 
     };
-}
\ No newline at end of file
+}
+
+#[no_mangle]
+pub extern "C" fn cudaMemcpyAsync(
```

Comment:
This is supposed to be an async function. I don't see the implementation is async?

Reply:
the function signature require one more parameter- cudaStream